src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -13731,6 +13810,71 @@ def _window_agg(
                     for quoted_identifier in frame.data_column_snowflake_quoted_identifiers
                 }
             ).frame
+        elif agg_func == "corr":
+            if window_func == WindowFunction.ROLLING and not isinstance(window, int):
+                ErrorMessage.not_implemented(f"window must be an integer value.")
```

Comment:
This is already covered by the above call to `check_and_raise_error_rolling_window_supported_by_snowflake` at the beginning of the function